api/queries_repo.go

```diff
@@ -738,53 +738,56 @@ func (m *RepoMetadataResult) LabelsToIDs(names []string) ([]string, error) {
 	return ids, nil
 }
 
-// ProjectsToIDs returns two arrays:
+// ProjectsTitlesToIDs returns two arrays:
```

Comment:
There's a lot of renaming to use the word "title" here to bring it into line with the `--projects` flag. Sorry if it's messy, the real changes are in `create` and `create_test`.

Reply:
Thanks, I saw the comment in reviewing the chain of _"builds on"_ PRs 👍